### DISCUSSION TITLE
there is no logo for kali linux ?

### DISCUSSION THREAD

**Discussion by HShazen** [89luca89/distrobox#1465] [Q&A] [answered]
why there is no icon for kali linux i mean it just the logo of distrobox and the logo of kali linux 
![kali-distrobox2](https://github.com/89luca89/distrobox/assets/139772928/6bb6a59a-9727-498a-a6b5-393a6efd22ee)

or 

![kali-distrobox](https://github.com/89luca89/distrobox/assets/139772928/c94cc8c9-5d13-4a03-814e-1f9d45553e71)

**Comment by 89luca89** [comment, marked as answer]
https://github.com/89luca89/distrobox/pull/1451

**Reply by HShazen** [reply]
Thank you.
but I wish that you have an answer to this question https://github.com/89luca89/distrobox/pull/1451#issuecomment-2211506725
